packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -501,7 +501,7 @@ export function unpackRuntimeMessage(message: ISequencedDocumentMessage) {
             message.type = innerContents.type;
             message.contents = innerContents.contents;
         }
-        assert(isRuntimeMessage(message), 0x122 /* "Message to unpack is not proper runtime message" */);
+        assert(isLegacyRuntimeMessage(message), 0x122 /* "Message to unpack is not proper runtime message" */);
```

Comment:
I do not think naming here is correct. It's correct at loader level, where we should see zero such events (or very close to zero).
Here - pretty much every session will see such events. The better name for this layer is "unpacked runtime message".
I.e. even if check is exactly the same, I'd rather use correct terminology by using functions with same bodies.

That said, they should not have same bodies.
"Legacy" should not include such things as "rejoin" & "alias" - these are pretty recent additions, they were added long after runtime started to wrap everything via "op", so we should never see these ops at loader layer.
Not sure about other types, but possible that couple other types are in the same bucket.

Reply:
Probably we can just change name isLegacyRuntimeMessage to isUnpackedRuntimeMessage.